src/componenets/calculator.js

```diff
@@ -1,27 +1,42 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
 function CalculatorStructure() {
+  const [state, setState] = useState({ total: 0, next: null, operation: null });
+  const setupEvent = (e) => {
+    const btnName = e.target.textContent;
+    const result = calculate(state, btnName);
+    setState(result);
+  };
+  const { total, next, operation } = state;
+
   return (
     <section className="calculatorContain">
-      <p className="displaySection">0</p>
+      <div className="displaySection">
+        {total}
+        {operation}
+        {next}
+      </div>
```

Comment:
[OPTIONAL] Great work done so far 💯  👍🏼 , I want just to mention a styling issue in this part this input is overflowing the calculator container, make sure to fixe it to make your calculator look good 

![Capture d’écran du 2023-07-15 17-42-28](https://github.com/geekelo/recalc-webapp/assets/59707859/ca30c184-7e5a-4953-90fb-6873e3652125)

Reply:
@souad988 thanks
The style error has been fixed